core/src/test/java/io/questdb/PropServerConfigurationTest.java

```diff
@@ -701,7 +701,7 @@ public void testSetAllFromFile() throws IOException, ServerConfigurationExceptio
             Assert.assertEquals(509, configuration.getCairoConfiguration().getMkDirMode());
             Assert.assertEquals(1000000, configuration.getCairoConfiguration().getParallelIndexThreshold());
             Assert.assertEquals(10, configuration.getCairoConfiguration().getReaderPoolMaxSegments());
-            Assert.assertEquals(5_000_000, configuration.getCairoConfiguration().getSpinLockTimeoutUs());
+            Assert.assertEquals(5_000_000, configuration.getCairoConfiguration().getSpinLockTimeout());
```

Comment:
5_000_000 should be 5_000?

Reply:
the value is not relevant, this test asserts that property can be read from file